### ISSUE TITLE
Question about training.

### ISSUE THREAD

**Issue by CVStack** [dahiyaaneesh/peclr#16]
Hi. 

Thank you for sharing the code.

I have a question about training the model in the finetunning stage.

I understood that the model uses the joint 2d and relative depth loss to train the hand pose estimator(pretrained encoder + fc layer), but there is other module to estimate the root depth and it seems necessary to train. 

But it is not possible to update that module with 2d joint and relative depth loss.

Is it right to use other loss function in addition to existing loss functions (ex> 3d joint loss) to update that module?

Thank you!
  
![image](https://user-images.githubusercontent.com/38209962/197337341-f5371d5c-3108-400a-af4c-f6a04a0577fd.png)

**Comment by CVStack** [comment]
https://github.com/dahiyaaneesh/peclr/issues/9
